examples/combined_ODNP.py

```diff
@@ -254,12 +251,6 @@
         logger.debug(
             "SETTING THIS POWER", this_dB, "(", dB_settings[j - 1], powers[j], "W)"
         )
-        if j == 0:
-            # why do we dip lock at the lowest power -- I don't quite understand this
-            retval = p.dip_lock(
-                config_dict['uw_dip_center_GHz'] - config_dict['uw_dip_width_GHz'] / 2,
-                config_dict['uw_dip_center_GHz'] + config_dict['uw_dip_width_GHz'] / 2,
-            )
```

Comment:
why are you removing this?

Reply:
elsewhere you say that you need to diplock at least once, and if that's the case, don't you need this?